### PR TITLE
[Admin] Edit voucher

### PR DESCRIPTION
**Pull request by DTHNhung**
## Related Tickets:

-   [#46552](https://edu-redmine.sun-asterisk.vn/issues/46552)

## WHAT this PR does?

-  Edit voucher in Admin
- Chưa bao gồm i18n js

### CODE REVIEW

**Comment by DTHNhung** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -41,6 +43,8 @@
     Route::get('/count-product', [CartController::class, 'countProduct'])->name('countProduct');
     Route::post('/add-more-product/{id}', [CartController::class, 'addMoreProduct'])->name('addMoreProduct');
     Route::post('/add-quantity/{id}', [CartController::class, 'addMoreProduct'])->name('addQuantity');
+    Route::post('/carts/apply-voucher', [CartController::class, 'applyVoucher'])->name('carts.apply.voucher');
+    Route::post('/carts/delete-voucher', [CartController::class, 'deleteVoucher'])->name('carts.delete.voucher');
```
Comment (huyenttm-0901) [inline routes/web.php]: delete thì dùng method delete nhé em

File: routes/web.php [hunk 2]
```diff
@@ -57,6 +61,16 @@
...
 });
 
+Route::group(['prefix' => 'admin/vouchers', 'middleware' => ['auth', 'admin']], function () {
+    Route::get('/', [VoucherController::class, 'index'])->name('vouchers.index');
+    Route::post('/', [VoucherController::class, 'store'])->name('vouchers.store');
+    Route::get('/list', [VoucherController::class, 'fetch'])->name('voucher.fetch');
+    Route::post('/edit', [VoucherController::class, 'edit'])->name('vouchers.edit');
+    Route::post('/update', [VoucherController::class, 'update'])->name('vouchers.update');
+    Route::post('/delete', [VoucherController::class, 'delete'])->name('vouchers.delete');
+    Route::post('/delete-list', [VoucherController::class, 'deleteList'])->name('vouchers.delete.list');
```
Comment (huyenttm-0901) [inline routes/web.php]: tương tự

File: routes/web.php [hunk 3]
```diff
@@ -57,6 +61,16 @@
     Route::resource('orders', OrderController::class)->only(['index', 'show', 'edit', 'update']);
 });
 
+Route::group(['prefix' => 'admin/vouchers', 'middleware' => ['auth', 'admin']], function () {
+    Route::get('/', [VoucherController::class, 'index'])->name('vouchers.index');
+    Route::post('/', [VoucherController::class, 'store'])->name('vouchers.store');
+    Route::get('/list', [VoucherController::class, 'fetch'])->name('voucher.fetch');
+    Route::post('/edit', [VoucherController::class, 'edit'])->name('vouchers.edit');
+    Route::post('/update', [VoucherController::class, 'update'])->name('vouchers.update');
```
Comment (huyenttm-0901) [inline routes/web.php]: update thì dùng patch nhá

**Comment by DTHNhung** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**